### PR TITLE
Updated to 2.0.0

### PR DESCRIPTION
**Pull request by ankurcha**
Please be aware that this change also removes the curl based user registration. This was more of a bugfixing thing but ended up removing behavior (not the intention). Also, having that breaks the scriptability of the `brew install tokumx-bin` command. 
I would want to recommend the repo owner to keep that in mind and probably come up with some other way of making people register - I just can't think of one at the moment.

### CODE REVIEW

File: tokumx-bin.rb [hunk 1]
```diff
@@ -2,23 +2,14 @@
...
   def install
-    email = `osascript -e 'Tell application "System Events" to display dialog "Provide your email address to keep up with TokuMX news:" default answer "email address"' -e "text returned of result"`
-    raise CannotInstallFormulaError, "Canceling at user request." unless $?.success?
-    email.strip!
-    unless email.empty?
-      curl "-X", "POST", "--data-urlencode", "email=#{email}", "--data-urlencode", "source=homebrew", "--data-urlencode", "product=tokumx", "-o", "/dev/null", "-s", "http://www.tokutek.com/simple_create_account.php"
-    else
-      curl "-X", "POST", "--data-urlencode", "email=anonymous@homebrew-installer.com", "--data-urlencode", "source=homebrew", "--data-urlencode", "product=tokumx", "-o", "/dev/null", "-s", "http://www.tokutek.com/simple_create_account.php"
-    end
-
```
Comment (leifwalsh) [inline tokumx-bin.rb:21]: Sorry, we need to keep this part.  What was the problem with curl?  It wasn't escaping the parentheses in the user agent string homebrew added?

Comment (ankurcha) [inline tokumx-bin.rb:21]: I kinda expected that. This was the error

```
Error: Failure while executing: /usr/bin/curl -fLA Homebrew\ 0.9.5\ (Ruby\ 2.0.0-481;\ Mac\ OS\ X\ 10.10) -X POST --data-urlencode email=anonymous@homebrew-installer.com --data-urlencode source=homebrew --data-urlencode product=tokumx -o /dev/null -s http://www.tokutek.com/simple_create_account.php

$ /usr/bin/curl -f#LA Homebrew\ 0.9.5\ (Ruby\ 2.0.0-481;\ Mac\ OS\ X\ 10.10) -X POST --data-urlencode email=achauhan@brightcove.com --data-urlencode source=homebrew --data-urlencode product=tokumx
bash: syntax error near unexpected token `('
```

Comment (leifwalsh) [inline tokumx-bin.rb:21]: Looks like a bug in homebrew.  Let me find a mac and try to reproduce it.

Comment (ankurcha) [inline tokumx-bin.rb:21]: Thanks. Let me know if you need some info from my end. [apple store near you](https://www.google.com/webhp?q=apple+store+near+me) - Store driven testing ;-)

Comment (leifwalsh) [inline tokumx-bin.rb:21]: Ok, never mind, this is an issue on the server side.  Website redesign must have messed something up.

**Comment by leifwalsh** [comment]
cherry picked the version and checksum changes, will continue working on the curl bit with the web server folks.

Thanks @ankurcha!

File: tokumx-bin.rb (hunk 1 above)
Comment (ankurcha) [inline tokumx-bin.rb:21]: oh cool. might I suggest adding the ability to pass the email as a commandline flag? `--email foo@bar.com` and making that a required thing. - to make scripting easier.

**Comment by ankurcha** [comment]
Thanks!

**Comment by leifwalsh** [comment]
That's a good idea, I'll work on it.

On Wed, Oct 1, 2014 at 3:58 PM, Ankur Chauhan notifications@github.com
wrote:

> Thanks!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Tokutek/homebrew-tokumx/pull/3#issuecomment-57528010.

## 

Cheers,
Leif

**Comment by ankurcha** [comment]
:+1:

**Comment by leifwalsh** [comment]
Should be done now, try it out.  Check README.md.

**Pull request closed without merging**